lib/PayPal/Auth/OAuthTokenCredential.php

```diff
@@ -143,7 +149,7 @@ public function getAccessToken($config)
 
             // Case where we have an old unencrypted cache file
             if (!array_key_exists('accessTokenEncrypted', $token)) {
-                AuthorizationCache::push($config, $this->clientId, $this->encrypt($this->accessToken), $this->tokenCreateTime, $this->tokenExpiresIn);
+                $AuthorizationCacheClass::push($config, $this->clientId, $this->encrypt($this->accessToken), $this->tokenCreateTime, $this->tokenExpiresIn);
```

Comment:
We need some checks to verify if the class is infact what we expect. Having an interface here would be perfect.